index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h3>Welcome</h3>
+      <img src="./images/Union.svg" alt="menu icon" />
+    </header>
+    <section class="headline">
+      <div class="whoiam">
+        <h2 class="name">Hey there. I'm Fatima Ezzahra</h2>
```

Comment:
- [ ] Due to the requirement of semantic HTML as mentioned above, kindly change this tag to `h1`